<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideShow</class>
 <widget class="QMainWindow" name="SlideShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QPushButton" name="previousButton">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="nextButton">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rotateLButton">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Rotate left</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rotateRButton">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>10</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 170, 255);</string>
       </property>
       <property name="text">
        <string>Rotate Right</string>
       </property>
      </widget>
      <widget class="QPushButton" name="zoomInButton">
       <property name="geometry">
        <rect>
         <x>830</x>
         <y>20</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Flick down to zoom in</string>
       </property>
      </widget>
      <widget class="QPushButton" name="zoomOutButton">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>20</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Flick up to zoom out</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exitButton">
       <property name="geometry">
        <rect>
         <x>1160</x>
         <y>550</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
      <widget class="QLabel" name="image">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>140</y>
         <width>551</width>
         <height>391</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 170, 255);
border-top-color: rgb(0, 170, 255);
border-left-color: rgb(0, 170, 255);
border-right-color: rgb(0, 170, 255);
border-bottom-color: rgb(0, 170, 255);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>60</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Swipe right</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>60</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Swipe left</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1252</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
